rai/node/node.cpp

```diff
@@ -2764,8 +2898,18 @@ void rai::election::confirm_once (MDB_txn * transaction_a)
 				BOOST_LOG (node.log) << boost::str (boost::format ("Retaining block %1%") % last_winner->hash ().to_string ());
 			}
 		}
+		else if (exceeded_min_threshold && (bool)last_winner_callback_info)
```

Comment:
Personally, I think we should only say a block was confirmed if it exceeded 50%, since that's the only safe number (and even then not completely safe due to reasons I won't write here).